lib/active_resource/connection_ext.rb

```diff
@@ -5,6 +5,15 @@ class Connection
 
     attr_reader :response
 
+    alias_method :raw_initialize, :initialize
```

Comment:
I am in favour. One other comment - I hit this error a lot in tests so maybe we should make a note about setting the shopify session for your test suite. You might even be able to make this extra message only show during tests